tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -360,29 +370,31 @@ class MklConvFwdPrimitive : public MklPrimitive {
 // TODO(nhasabni): We should not require passing a type to MklPrimitiveFactory.
 // But removing the need for type in MklPrimitiveFactory is going to require
 // change to every MKL op. So not doing it now. Instead passing float.
-template <typename Tinput, typename Tfilter, typename Tbias, typename Toutput>
+template <typename Tinput, typename Tfilter, typename Tbias, typename Toutput,
+          bool native_format>
```

Comment:
Same. Can we make `native_format` a normal parameter?

Reply:
Done.